crates/cast/tests/data/scripts/hello_world/hello_world.cairo

```diff
@@ -0,0 +1,65 @@
+use array::ArrayTrait;
+use array::SpanTrait;
+use traits::Into;
+use traits::TryInto;
+use serde::Serde;
+use traits::Drop;
+
+use debug::PrintTrait;
+
+use starknet::{
+    testing::cheatcode, ClassHash, ContractAddress, ClassHashIntoFelt252,
+    ContractAddressIntoFelt252, Felt252TryIntoClassHash, Felt252TryIntoContractAddress,
+    contract_address_const
+};
+
+#[derive(Drop, Clone)]
+struct CallResult {
+    data: Array::<felt252>,
+}
+
+fn call(
+    contract_address: ContractAddress, function_name: felt252, calldata: Array::<felt252>
+) -> CallResult {
```

Comment:
In the next iteration, the `call` function will become importable from a separate module + more tests for deployment scripts with `call` will be added.

Reply:
I guess this is this: https://github.com/foundry-rs/starknet-foundry/pull/1111

Reply:
#1111 merged to simplify rebasing